articles/client_configuration.md

```diff
@@ -56,6 +56,10 @@ The code block below displays an example config file with the available paramete
 
 <br/>
 
+## Create a .synapseConfig file on programmatic client installation
+
+When you install the [Synapse Python client](http://docs.synapse.org/python/#installation), a `.synapseConfig` file filled with examples, much like the sample above, should appear in your home directory if a config file is not already present. You can uncomment and fill out the parameters of interest to you to use. 
```

Comment:
What happens when you install `synapser`?

Reply:
change 'if a config file' to 'if a configuration file'

'of interest *for* you to use'

Reply:
Hmm not sure about the behavior of synapser but I'll try it out and edit the test to say yay or nay.